pkg/prometheus/metrics.go

```diff
@@ -77,3 +103,7 @@ func RecordFormPasswordAuth(result string) {
 	authFormCounter.Inc()
 	authFormCounterResult.WithLabelValues(result).Inc()
 }
+
+func X509MissingSANMetric(p Provider) metrics.CounterMetric {
+	return x509MissingSANCounter.WithLabelValues(string(p))
+}
```

Comment:
inline

Reply:
ack, let's do, the function is probably not worth it.